src/frontend/js/components/UserMenu/DesktopUserMenu.tsx

```diff
@@ -42,9 +43,7 @@ export const DesktopUserMenu: FC<UserMenuProps> = ({ user }) => {
       </label>
       <button {...getToggleButtonProps()} className="selector__button">
         {user.username}
-        <svg role="img" className="selector__button__icon" aria-hidden="true">
-          <use xlinkHref="#icon-chevron-down" />
-        </svg>
+        <Icon name="icon-chevron-down" className="selector__button__icon" />
```

Comment:
This icon has now the `.icon` className applied but it introduces a visual regression.

<img width="71" alt="image" src="https://user-images.githubusercontent.com/9265241/155690385-f684ec23-179e-4d50-88e5-52338eb5ddab.png">